### ISSUE TITLE
JS / CSS missing in current bundle

### ISSUE THREAD

**Issue by davidak** [apertus-open-source-cinema/elmyra#8]
```
127.0.0.1 - - [17/Feb/2016 17:22:15] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [17/Feb/2016 17:22:15] "GET /static/css/styles.css HTTP/1.1" 404 -
127.0.0.1 - - [17/Feb/2016 17:22:15] "GET /static/js/scripts.js HTTP/1.1" 404 -
```

tested with OS X and Linux.

**Comment by simonrepp** [comment]
Oh great :) (the report, not the circumstances) ... I'll look into it. Thanks a lot!

**Comment by simonrepp** [comment]
It's some concurrency mess with gulp. (I feared that a little haha)

**Comment by simonrepp** [comment]
Should be fixed in 2f2b74abd8a5b14c34bced7e857670870483ae57; I don't feel super confident about the resolution, but at least right now I couldn't produce any new concurrency issues in multiple tests.

The updated release bundles will be up in < 2 minutes.

Thanks again for the report david :+1:
